wiki/History_of_osu!/Online_rankings/de.md

```diff
@@ -9,6 +9,9 @@ tags:
   - Geschichte
   - erster Rang
   - bester Spieler
+outdated: true
```

Comment:
this line isn't needed in any of the articles you added it to, only `outdated_since` and `outdated_translation`